CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.10.17-dev3
+
+### Enhancements
+
+* **Improves salesforce partitioning** Partitions Salesforce data as xlm instead of text for improved detail and flexibility. Partitions htmlbody instead of textbody for Salesforce emails. Importance: Allows all Salesforce fields to be ingested and gives Salesforce emails more detailed partitioning.
+
+### Features
+
+### Fixes
```

Comment:
description here is good, but just need to bump the 0.10.17-dev2 below to 0.10.17-dev3 and include this enhancement note in that existing section

Reply:
@potter-potter bump ^ (also probably need to merge up now)